### ISSUE TITLE
[REQUEST] Finisher Critical

### ISSUE THREAD

**Issue by mmateusu** [mikedg1/Fire-Pro-World-Mod-Requests-and-Sample-Mod#3]
Is there any way to do a mod that makes the Finisher/Signatures always criticals?

BTW: Don't know how to post here bc there isn't any requests before.

**Comment by mikedg1** [comment]
This was a perfect post, I'll look into it. This should be doable.

**Comment by mmateusu** [comment]
Can you do it?

**Comment by mikedg1** [comment]
Yeah, looks like I should be able to , just don't have a timeline yet.
